pylintrc

```diff
@@ -0,0 +1,38 @@
+[MASTER]
+# A comma-separated list of package or module names from where C extensions may
+# be loaded. Extensions are loading into the active Python interpreter and may
+# run arbitrary code
+extension-pkg-whitelist=numpy
+indent-string="  "
```

Comment:
nit: if you're gonna keep the [MASTER], etc. headings, indent-string goes under [FORMAT]. Or just remove the headings. Also, please put a blank line between the different directives.